<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialogClass</class>
 <widget class="QDialog" name="MainDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>647</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yu Gothic UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>FileTimeChanger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="Label_FileListCnt">
       <property name="text">
        <string>ファイルリスト (0/0)(&amp;L):</string>
       </property>
       <property name="buddy">
        <cstring>ListView_Files</cstring>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="ListView_Files"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_SelectFiles">
       <property name="text">
        <string>選択...(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_AddFile">
       <property name="text">
        <string>追加...(&amp;A)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_DeleteFile">
       <property name="text">
        <string>削除(&amp;D)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OpenDir">
       <property name="text">
        <string>場所を開く...(&amp;F)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="RadioButton_AllFileProp">
         <property name="text">
          <string>すべてのファイルを同一の日時に変更する</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RadioButton_OneFileProp">
         <property name="text">
          <string>ファイルごとに別々の日時に変更する</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日時プロファイル</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="CheckButton_CheckDateTime">
             <property name="text">
              <string>作成日時(&amp;C):</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDateEdit" name="DateEdit_CreateDate"/>
           </item>
           <item>
            <widget class="QTimeEdit" name="TimeEdit_CreateTime"/>
           </item>
           <item>
            <widget class="QPushButton" name="Button_CreateNow">
             <property name="text">
              <string>現在日時に</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Create_Create">
             <property name="text">
              <string>別の日時</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Create_Update">
             <property name="text">
              <string>更新日時と合わせ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Create_Access">
             <property name="text">
              <string>ｱｸｾｽ日時と合わせ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QCheckBox" name="CheckButton_UpdateDateTime">
             <property name="text">
              <string>更新日時(&amp;U):</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDateEdit" name="DateEdit_UpdateDate"/>
           </item>
           <item>
            <widget class="QTimeEdit" name="TimeEdit_UpdateTime"/>
           </item>
           <item>
            <widget class="QPushButton" name="Button_UpdateNow">
             <property name="text">
              <string>現在日時に</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Update_Update">
             <property name="text">
              <string>別の日時</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Update_Create">
             <property name="text">
              <string>作成日時と合わせ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Update_Access">
             <property name="text">
              <string>ｱｸｾｽ日時と合わせ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="CheckButton_AccessDateTime">
             <property name="text">
              <string>ｱｸｾｽ日時(&amp;E):</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QDateEdit" name="DateEdit_AccessDate"/>
           </item>
           <item>
            <widget class="QTimeEdit" name="TimeEdit_AccessTime"/>
           </item>
           <item>
            <widget class="QPushButton" name="Button_AccessNow">
             <property name="text">
              <string>現在日時に</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Access_Access">
             <property name="text">
              <string>別の日時</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Access_Create">
             <property name="text">
              <string>作成日時と合わせ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="RadioButton_Access_Update">
             <property name="text">
              <string>更新日時と合わせ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>DateEdit_CreateDate</zorder>
     <zorder>TimeEdit_CreateTime</zorder>
     <zorder>DateEdit_UpdateDate</zorder>
     <zorder>TimeEdit_UpdateTime</zorder>
     <zorder>DateEdit_AccessDate</zorder>
     <zorder>TimeEdit_AccessTime</zorder>
     <zorder>checkBox_4</zorder>
     <zorder>checkBox_5</zorder>
     <zorder>line_2</zorder>
     <zorder>line_3</zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder></zorder>
     <zorder>frame</zorder>
     <zorder>frame_2</zorder>
     <zorder>frame_3</zorder>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="CheckButton_TransformFolderAlso">
         <property name="text">
          <string>フォルダーの日時を変更する</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CheckButton_TransformRecursion">
         <property name="text">
          <string>フォルダー内のファイルの日時も変更する</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Transform">
       <property name="text">
        <string>変更(&amp;T)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel">
       <property name="text">
        <string>キャンセル(&amp;X)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>Button_SelectFiles</zorder>
  <zorder>Button_Transform</zorder>
  <zorder>Button_Cancel</zorder>
  <zorder>ListView_Files</zorder>
  <zorder>line</zorder>
  <zorder>groupBox</zorder>
  <zorder>line</zorder>
  <zorder>Button_AddFile</zorder>
  <zorder>Button_DeleteFile</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>RadioButton_AllFileProp</zorder>
  <zorder>RadioButton_OneFileProp</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_4</zorder>
  <zorder>RadioButton_Create_Create</zorder>
  <zorder>RadioButton_Create_Update</zorder>
  <zorder>RadioButton_Create_Access</zorder>
  <zorder>RadioButton_Update_Access</zorder>
  <zorder>RadioButton_Update_Update</zorder>
  <zorder>RadioButton_Update_Create</zorder>
  <zorder>RadioButton_Access_Create</zorder>
  <zorder>RadioButton_Access_Access</zorder>
  <zorder>RadioButton_Access_Update</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>CheckButton_TransformRecursion</zorder>
  <zorder>CheckButton_TransformFolderAlso</zorder>
  <zorder>horizontalSpacer_9</zorder>
  <zorder>horizontalSpacer_10</zorder>
  <zorder>Label_FileListCnt</zorder>
  <zorder>horizontalSpacer_11</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>ListView_Files</tabstop>
  <tabstop>Button_SelectFiles</tabstop>
  <tabstop>Button_AddFile</tabstop>
  <tabstop>Button_DeleteFile</tabstop>
  <tabstop>Button_OpenDir</tabstop>
  <tabstop>RadioButton_AllFileProp</tabstop>
  <tabstop>RadioButton_OneFileProp</tabstop>
  <tabstop>CheckButton_CheckDateTime</tabstop>
  <tabstop>DateEdit_CreateDate</tabstop>
  <tabstop>TimeEdit_CreateTime</tabstop>
  <tabstop>RadioButton_Create_Create</tabstop>
  <tabstop>RadioButton_Create_Update</tabstop>
  <tabstop>RadioButton_Create_Access</tabstop>
  <tabstop>CheckButton_UpdateDateTime</tabstop>
  <tabstop>DateEdit_UpdateDate</tabstop>
  <tabstop>TimeEdit_UpdateTime</tabstop>
  <tabstop>RadioButton_Update_Update</tabstop>
  <tabstop>RadioButton_Update_Create</tabstop>
  <tabstop>RadioButton_Update_Access</tabstop>
  <tabstop>CheckButton_AccessDateTime</tabstop>
  <tabstop>DateEdit_AccessDate</tabstop>
  <tabstop>TimeEdit_AccessTime</tabstop>
  <tabstop>RadioButton_Access_Access</tabstop>
  <tabstop>RadioButton_Access_Create</tabstop>
  <tabstop>RadioButton_Access_Update</tabstop>
  <tabstop>CheckButton_TransformFolderAlso</tabstop>
  <tabstop>CheckButton_TransformRecursion</tabstop>
  <tabstop>Button_Transform</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="MainDialog.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Create_Create</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>393</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Create_Update</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Create_Access</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Update_Update</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>467</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Update_Create</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Update_Access</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Access_Access</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>510</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Access_Create</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>RadioButton_Access_Update</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>DateEdit_AccessDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>504</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>TimeEdit_AccessTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>DateEdit_UpdateDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>TimeEdit_UpdateTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>TimeEdit_CreateTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>DateEdit_CreateDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_CheckDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>Button_CreateNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_UpdateDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>Button_UpdateNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckButton_AccessDateTime</sender>
   <signal>clicked(bool)</signal>
   <receiver>Button_AccessNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>500</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
